.github/workflows/ci_run_scm_DEPHY.yml

```diff
@@ -54,6 +54,18 @@ jobs:
     #######################################################################################
     # Install FORTRAN dependencies
     #######################################################################################
+    - name: Install openmpi
```

Comment:
Same here, are we sure the appropriate compilers are being used?

Reply:
For all these tests, openmpi is only used to build the SCM, while gfortran is used to build the other dependencies. We aren't exercising the MPI functionality, so it works fine, but maybe not the cleanest. 
We could move the FC/CC declarations and build everything with mpif90. (Maybe wait until after the ccpp library dependency removal PR goes in, where we will remove a big chunk of this script?)